### ISSUE TITLE
listDrivers() err

### ISSUE THREAD

**Issue by Lagikna** [feihoo87/QuLab#11]
'''python
AttributeError                            Traceback (most recent call last)
<ipython-input-8-9b4b39d767dc> in <module>()
----> 1 listDrivers()

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab\_bootstrap.py in listDrivers()
    141 def listDrivers():
    142     ret = _schema.Driver.objects()
--> 143     ui.list_drivers(ret)
    144 
    145 def listInstruments():

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab\ui\__init__.py in list_drivers(drivers)
     47     for driver in drivers:
     48         module = driver.module
---> 49         table.append('%s|%s|%s|%s' % (driver.name, driver.version, module.fullname, module.created_time))
     50         if not module.is_package:
     51             continue

AttributeError: 'NoneType' object has no attribute 'fullname'
'''

**Comment by feihoo87** [comment]
你的Markdown写得不对啊，提交前先 Preview 一下。
```
代码块是用键盘左上角的 ` 而不是单引号 ' 
```

**Comment by Lagikna** [comment]
```python
wo fa xian le 
```
listDrivers这个怎么解决？

**Comment by Lagikna** [comment]
```python
AttributeError Traceback (most recent call last)
in ()
----> 1 listDrivers()

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab_bootstrap.py in listDrivers()
141 def listDrivers():
142 ret = _schema.Driver.objects()
--> 143 ui.list_drivers(ret)
144
145 def listInstruments():

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab\ui_init_.py in list_drivers(drivers)
47 for driver in drivers:
48 module = driver.module
---> 49 table.append('%s|%s|%s|%s' % (driver.name, driver.version, module.fullname, module.created_time))
50 if not module.is_package:
51 continue

AttributeError: 'NoneType' object has no attribute 'fullname'
```

**Comment by feihoo87** [comment]
评论中出现这样的

feihoo87 closed this in 38cf21d 3 minutes ago

就说明我 commit 了新代码，点进去可以看修改内容。你再更新本地代码测试一下。

**Comment by Lagikna** [comment]
## 还有点问题
```py
TypeError                                 Traceback (most recent call last)
<ipython-input-3-9b4b39d767dc> in <module>()
----> 1 listDrivers()

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab\_bootstrap.py in listDrivers()
    141 def listDrivers():
    142     ret = _schema.Driver.objects()
--> 143     ui.list_drivers(ret)
    144 
    145 def listInstruments():

B:\WinPython-64bit-3.6.3.0Qt5\python-3.6.3.amd64\lib\site-packages\lab\ui\__init__.py in list_drivers(drivers)
     48         module = driver.module
     49         if module is None:
---> 50             table.append('%s|%s| Error! Module for driver not set |N/A' % driver.name, driver.version)
     51             continue
     52         table.append('%s|%s|%s|%s' % (driver.name, driver.version, module.fullname, module.created_time))

TypeError: not enough arguments for format string
```
